inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleTableSourceFactory.java

```diff
@@ -127,7 +118,17 @@ public DynamicTableSource createDynamicTableSource(Context context) {
                 startupOptions,
                 sourceMultipleEnable,
                 inlongMetric,
-                inlongAudit);
+                inlongAudit,
```

Comment:
There are too many parameters here. If you add a parameter later, you need to modify the constructor. It is recommended to abstract a data model and encapsulate it.

Reply:
It is a good suggestion, but there is not much time now.I will optimize it in next pr.